### PR TITLE
DevWs for RoomsBeautiful containing containing Curtain

### PR DESCRIPTION
**Pull request by devropes**

### CODE REVIEW

**Comment by devropes** [comment]
The only change I made is vid: "generic-shade-2" to display the shade level control.

File: devicetypes/rooms-beautiful/rooms-beautiful-curtain.src/rooms-beautiful-curtain.groovy [hunk 1]
```diff
@@ -136,7 +136,7 @@ def parse(String description) {
                 }
                 break;
             case 0xFC00:
-                if (description?.startsWith('read attr -'))
+                if (description ? .startsWith('read attr -'))
```
Comment (greens) [inline devicetypes/rooms-beautiful/rooms-beautiful-curtain.src/rooms-beautiful-curtain.groovy]: I just checked this in the interpreter to be sure, but this won't run.

Interestingly, `description ?. startsWith...` does, but we'd prefer `description?.startsWith`

Comment (devropes) [reply devicetypes/rooms-beautiful/rooms-beautiful-curtain.src/rooms-beautiful-curtain.groovy]: I have now fixed that. Must've happened when I was using a code beautifier to fix the indentation.

**Review by greens — APPROVED** [review]

**Comment by greens** [comment]
should be live on the 25th in production

**Pull request merged**